static/js/tools/choropleth/choropleth.tsx

```diff
@@ -129,6 +142,21 @@ class ChoroplethMap extends Component<PropsType, StateType> {
     );
   };
 
+  /**
+   * Gets the max date in a list of ISO 8601 dates.
+   * @param dates: a list of dates in ISO 8601 format.
+   * Example: "2020-01-02".
+   */
+  public getMaxDate = (dates: string[]): string => {
+    if (!dates.length) {
+      return "";
+    }
+
+    return dates.reduce((a, b) => {
+      return new Date(a) > new Date(b) ? a : b;
```

Comment:
Is > or >= the proper operator?  It depends on the reduce specification.

Reply:
should be >= i believe, thanks. fixed.